pkg/controller/operands/networkAddons.go

```diff
@@ -61,12 +61,24 @@ func (h *cnaHooks) updateCr(req *common.HcoRequest, Client client.Client, exists
 		return false, false, errors.New("can't convert to CNA")
 	}
 
+	// If deployOVS annotation doesn't exists prior the upgrade - set this annotation to true;
+	// Otherwise - remain the value as it is.
+	if req.UpgradeMode {
+		_, exists := req.Instance.Annotations["deployOVS"]
+		if !exists {
+			req.Instance.Annotations = map[string]string{}
```

Comment:
The current code removed all the existing annotations in case the `deployOVS` annotation is missing. Is this on purpose?

Also, please set `req.Dirty = true`. That will make sure the change is written back to the cluster.

Reply:
Thanks, fixed.